GeneralsMD/Code/GameEngine/Include/GameLogic/AI.h

```diff
@@ -420,7 +420,7 @@ struct AICommandParms
   Object*									m_obj;
   Object*									m_otherObj;
   const Team*							m_team;
-	std::vector<Coord3D>		m_coords;
+	mutable std::vector<Coord3D>		m_coords;
```

Comment:
After thinking about this more, let us do a lightweight refactor and only touch where we can easily avoid a copy without const_casts or mutables.

Then also close #1924

Reply:
Remove the mutable and then keep the copies where it would fail to compile.

Reply:
Ok I closed 1924 and AICommandParms::m_coords is now non-mutable again.